src/client/automation/index.js

```diff
@@ -49,7 +49,7 @@ exports.get = require;
 
 hammerhead.nativeMethods.objectDefineProperty.call(window, window, '%testCafeAutomation%', {
     enumerable:   false,
-    configurable: false,
+    configurable: true,
```

Comment:
Flags `enumerable`, `configurable` and `writable` have `false` value by default.
You can omit flags that has 'false' value.